### PR TITLE
fix: pluralization not being used for duration translations

### PR DESCRIPTION
**Pull request by kbond**
Pluralization is not being triggered for the new duration translations:

```twig
{{ 1|duration }} {# 1 second|%count% seconds #}
{{ 2|duration }} {# 2 seconds #}
```

I can't immediately see the issue...

### CODE REVIEW

**Comment by weaverryan** [comment]
You mean it's literally outputting `1 second|%count% seconds`?

**Comment by kbond** [comment]
> You mean it's literally outputting 1 second|%count% seconds?

Yes

**Comment by weaverryan** [comment]
What does `$seconds` look like by the time it gets here? https://github.com/KnpLabs/KnpTimeBundle/blob/main/src/DateTimeFormatter.php#L56

**Comment by kbond** [comment]
`1.0`, `2.0`, etc

**Comment by kbond** [comment]
Found the problem, I'm not passing %count% to https://github.com/KnpLabs/KnpTimeBundle/blob/3fe79604a94c0f77fc28613e9fc45b6ce665ecf6/src/DateTimeFormatter.php#L76

File: tests/IntegrationTest.php [hunk 1]
```diff
@@ -25,20 +25,20 @@ public function testServiceWiring(): void
...
-        $this->assertStringContainsString('One minute: 1 minute', $result);
-        $this->assertStringContainsString('Multiple minutes: 59 minutes', $result);
-        $this->assertStringContainsString('One hour: 1 hour', $result);
-        $this->assertStringContainsString('Multiple hours: 23 hours', $result);
-        $this->assertStringContainsString('One day: 1 day', $result);
-        $this->assertStringContainsString('Multiple days: 99 days', $result);
+        $this->assertStringContainsString("Yesterday: 1 day ago\n", $result);
+        $this->assertStringContainsString("Now: now\n", $result);
+
+        $this->assertStringContainsString("1 day ago\n", $result);
```
Comment (weaverryan) [inline tests/IntegrationTest.php:31]: this is a duplicate - and doesn't correspond to its own line in the template: https://github.com/KnpLabs/KnpTimeBundle/blob/main/tests/fixtures/template.twig

File: tests/IntegrationTest.php [hunk 2]
```diff
@@ -25,20 +25,20 @@ public function testServiceWiring(): void
...
+        $this->assertStringContainsString("Zero: < 1 second\n", $result);
+        $this->assertStringContainsString("Less than a second: < 1 second\n", $result);
+        $this->assertStringContainsString("One second: 1 second\n", $result);
+        $this->assertStringContainsString("Multiple seconds: 59 seconds\n", $result);
+        $this->assertStringContainsString("One minute: 1 minute\n", $result);
```
Comment (weaverryan) [inline tests/IntegrationTest.php:36]: What about 61 seconds? Does that still say 1 minute? Same for 1 hour and 1 minute (e.g. 3660 seconds)

File: tests/IntegrationTest.php (hunk 1 above)
Comment (kbond) [reply tests/IntegrationTest.php]: Split the tests

File: tests/IntegrationTest.php (hunk 2 above)
Comment (kbond) [reply tests/IntegrationTest.php:42]: > Does that still say 1 minute?

Yes, I added some tests for this.

**Comment by weaverryan** [comment]
Thanks Kevin!

**Pull request merged**